optimum/exporters/tasks.py

```diff
@@ -105,6 +105,8 @@ class TasksManager:
             "audio-ctc": "AutoModelForCTC",
             "audio-xvector": "AutoModelForAudioXVector",
             "stable-diffusion": "StableDiffusionPipeline",
+            "visual-question-answering": "AutoModelForVisualQuestionAnswering",
```

Comment:
```suggestion
            "visual-question-answering": "AutoModelForQuestionAnswering",
```

following https://huggingface.co/uclanlp/visualbert-vqa